php/BUILDING.md

```diff
@@ -3,16 +3,14 @@
 This page describes how to build and install Ice for PHP from source.
 
 ZeroC provides [binary distributions] for many platforms, including
-Linux and Windows, so building Ice for PHP from source is usually
-unnecessary.
+Linux, so building Ice for PHP from source is usually unnecessary.
 
 * [PHP Build Requirements](#php-build-requirements)
 * [Building the PHP Extension](#building-the-php-extension)
   * [Linux or macOS](#linux-or-macos)
   * [Windows](#windows)
```

Comment:
This should be removed from the toc

Reply:
GitHub highlighted the wrong line, I was trying to show `* [Windows](#windows)` in the table of contents.